drivers/tee/optee/rpmb.c

```diff
@@ -122,6 +122,10 @@ static u32 rpmb_process_request(struct optee_private *priv, void *req,
 {
 	struct rpmb_req *sreq = req;
 	struct mmc *mmc;
+	size_t rpmb_frame_data_sz;
```

Comment:
minor: for consistency I would rename this `rpmb_data_sz` and move this line below line 127.